content/pages/meetup/2024/1023-nycu.rst

```diff
@@ -0,0 +1,147 @@
+========================================
+Meetup 2024 October 23th in NYCU
+========================================
+
+:date: 2024-10-18 17:25
+:url: meetup/2024/1023-nycu
+:save_as: meetup/2024/1023-nycu.html
+
+Date
+-----
+
+* Date: October 16th, 2024
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+There will be many different subjects discussed at the same time.
+
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
+| Timetable       | Topic 1                  | Topic 2                                               | Topic 3                                      |
++=================+==========================+=======================================================+==============================================+
+| 18:30 ~ 19:30   | free chat                                                                                                                       |
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
+| 19:30 ~ 20:00   |                          | modmesh discussion and training :                     | nsd project presentation:                    |
+|                 | free chat                |                                                       |                                              |
+|                 |                          | 1. Use Bezier curve to fit NACA 4-digit airfoil       | 1. term project (PolygonDust) progress       |
+|                 |                          | 2. Prototype profiler object serialization using JSON |    sharing By Han-Xuan Huang.                |
+|                 |                          | 3. Build modmesh                                      | 2. term project (pybezier) progress          |
+|                 |                          | 4. Make a simple PR for modmesh                       |    sharing by Wei-Syuan Teng.                |
++-----------------+                          +                                                       +----------------------------------------------+
+| 20:00 ~ 21:00   |                          |                                                       | nsd homework assistance                      |
+|                 |                          |                                                       |                                              |
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
+| 21:00 ~ 21:30   | free chat                                                                                                                       |                            
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
```

Comment:
According to @yungyuc's suggestion in pr #482, I schedule the presetation time in the table.